src/main/resources/+ciplugins/+jenkins/getDefaultPlugins.m

```diff
@@ -6,9 +6,15 @@
     pluginProviderData (1,1) struct = struct();
 end
 
+if isMATLABReleaseOlderThan("R2025b")
+    reportPlugin = ciplugins.jenkins.BuildReportPlugin();
+else
+    reportPlugin = ciplugins.jenkins.ParallelizableBuildReportPlugin();
+end
```

Comment:
`buildReportPlugin` in place of `reportPlugin`? The latter feels slightly ambiguous though I'm fine with any considering the confusion it may create with existing `BuildReportPlugin`.